dm/master/server.go

```diff
@@ -458,7 +467,12 @@ func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.S
 				s.removeMetaLock.Unlock()
 				return resp, nil
 			}
-			err = s.removeMetaData(ctx, cfg)
+			toDB := cfg.TargetDB
+			toDB.Adjust()
+			if len(toDB.Password) > 0 {
+				toDB.Password = utils.DecryptOrPlaintext(toDB.Password)
+			}
+			err = s.removeMetaData(ctx, cfg.Name, cfg.MetaSchema, *toDB)
```

Comment:
The early change here is due to the need to call the RemoveMeta method when creating a task via the API later
https://github.com/Ehco1996/dm/blob/7c01c9cacf60d882548e0bbcc2486d444c9bace2/dm/master/openapi.go#L339

Reply:
DecryptOrPlaintext can still moved inside removeMetaData

Reply:
can we move `DecryptOrPlaintext` into `DBConfig.Adjust` ?